cms/templates/js/move-xblock-list.underscore

```diff
@@ -27,7 +33,7 @@
                         </span>
                     <% } %>
                     <span class="icon fa fa-arrow-right forward-sr-icon" aria-hidden="true"></span>
-                    <span class="sr forward-sr-text"><%- forwardButtonSRText %></span>
+                    <span class="sr forward-sr-text"><%- gettext("Click for children") %></span>
```

Comment:
@cptvitamin I understand the desire for conciseness, but I share your concern about how understandable "children" will be -- especially in translated instances. I really think it's preferable to use "child" as an adjective rather than "children" as a noun since we aren't talking about small humans. @andy-armstrong or @muhammad-ammar - do you have any ideas for a shorter alternative to "components" that can be used at any level?